.github/workflows/tests.yml

```
@@ -11,6 +11,32 @@ on:
       - 'docs/**'
 
 jobs:
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version:
+          - '3.10'
+    name: Windows, SQLite, Python ${{ matrix.python-version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Get pip cache dir
+        id: pip-cache
+        run: echo "::set-output name=dir::$(pip cache dir)"
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('tests/requirements/py3.txt') }}
+      - run: pip install -r tests/requirements/py3.txt -e .
+      - name: Run tests
+        run: python tests/runtests.py -v2
```

Let's use long flags as they're self-documenting:
```suggestion
      - run: pip install --requirement=tests/requirements/py3.txt --editable=.
      - name: Run tests
        run: python tests/runtests.py --verbosity=2
```

---

IMO we current version is fine, we use these shortcuts everywhere in docs.

---

I have to admit I had to go and test this command. I thought what does version 2 mean? 🤦 (I quickly realised my error!)